hal/src/tron/network/network.cpp

```diff
@@ -111,7 +112,13 @@ int if_init_platform(void*) {
     CHECK(hal_get_mac_address(HAL_DEVICE_MAC_ETHERNET, mac, HAL_DEVICE_MAC_ADDR_SIZE, nullptr));
 
     if (HAL_Feature_Get(FEATURE_ETHERNET_DETECTION)) {
-        en2 = new WizNetif(HAL_SPI_INTERFACE1, D5, D3, D4, mac);
+        WizNetifConfigData wizNetifConfigData;
+        WizNetifConfig::instance()->getConfigData(&wizNetifConfigData);
+        hal_pin_t cs_pin = (wizNetifConfigData.cs_pin == PIN_INVALID) ? D5 : wizNetifConfigData.cs_pin;
```

Comment:
Might be good to innitialize the config data with default pins, so that it's easy for us to change the default pins for all platforms, intead of traversing diffeerent folders to modify platform-specific `network.cpp`.

Reply:
This will only be for P2 for the time being.

Reply:
As per our team discussion this morning we will release for all platforms that support Ethernet.